addons/binding/org.openhab.binding.mihome/META-INF/MANIFEST.MF

```diff
@@ -8,10 +8,12 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.openhab.binding.mihome;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
-Export-Package: 
```

Comment:
The build is failing, maybe these spaces were required?

Reply:
That was it. Thanks for your help. 

Fixed the issue and pushed updated code.